hydra-node/src/Hydra/Network.hs

```diff
@@ -96,7 +97,7 @@ instance ToCBOR NodeId where
 -- REVIEW(SN): This is also used in hydra-tui
 data Host = Host
   { hostname :: Text
-  , port :: PortNumber
+  , port :: Port
```

Comment:
Should: keep `PortNumber` as the data type we use everywhere

`PortNumber` is a dedicated and more suitable type than `type Port = Int` for describing port numbers. We should rather just convert to `Int` when we need it.